packages/common/src/intl/locales/ar/app.json

```diff
@@ -1,15 +1,15 @@
 {
-  "app.admin": "مشرف",
-  "app.customer-requisition": "طلبات الشراء",
-  "app.dashboard": "لوحة القيادة",
-  "app.distribution": "توزيع",
-  "app.logout": "تسجيل خروج",
-  "app.messages": "رسائل",
-  "app.outbound-shipment": "شحنة صادرة",
-  "app.reports": "التقارير",
-  "app.stock": "الأوراق المالية",
-  "app.suppliers": "الموردين",
-  "app.sync": "تزامن",
-  "app.tools": "أدوات",
-  "app.welcome": "أهلا،"
+  "admin": "مشرف",
```

Comment:
thought that since these are in the 'app' namespace, we don't need to prefix them as well